osu.Game/Configuration/OsuConfigManager.cs

```diff
@@ -121,6 +121,7 @@ protected override void InitialiseDefaults()
             SetDefault(OsuSetting.PositionalHitsoundsLevel, 0.2f, 0, 1);
             SetDefault(OsuSetting.DimLevel, 0.7, 0, 1, 0.01);
             SetDefault(OsuSetting.BlurLevel, 0, 0, 1, 0.01);
+            SetDefault(OsuSetting.EditorDim, 0.25f);
```

Comment:
Where does this default come from? The previous was 66%.

Reply:
Also please add the full range and precision similar to `DimLevel`, and probably rename to `EditorDimLevel` to match.

Reply:
https://github.com/ppy/osu/pull/20907#issuecomment-1289946539
I mentioned the problem with default values here, I decided to use 25% through my visual observation.
And if you use 66% as the default value, the options don't exist.

I would choose to set it to 75% so it should be fine?

and [EditorWaveformOpacity](https://github.com/ppy/osu/blob/cb51fc7384ba687be486bede56bd603bc8febf66/osu.Game/Configuration/OsuConfigManager.cs#L174) also need full range and precision.
i can fix it by this time

Reply:
I think 75% is fine as a default.

I'm also not sure 100% dim should be exposed through the editor. This is not present in the stable editor and I think it will lead to beatmappers incorrectly considering the full spectrum of players. Maybe 75% can be the highest setting?